frontend/src/views/CellGuide/components/CellGuideCard/components/Description/index.tsx

```diff
@@ -58,7 +58,7 @@ interface DescriptionProps {
   cellTypeName: string;
   cellTypeId: string;
   skinnyMode: boolean;
-  setTooltipContent: Dispatch<
+  setTooltipContent?: Dispatch<
```

Comment:
This is only necessary for CellGuide as it handles tooltip display in mobile view, which isn't supported in WMG.